src/bin/vip-dev-environment-info.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @fomat
+ */
+
+/**
+ * External dependencies
+ */
+import debugLib from 'debug';
+import chalk from 'chalk';
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import { defaults, printEnvironmentInfo } from 'lib/dev-environment';
+import { DEV_ENVIRONMENT_COMMAND } from 'lib/constants/dev-environment';
+
+const debug = debugLib( '@automattic/vip:bin:dev-environment' );
+
+// Command examples
+const examples = [
+	{
+		usage: `${ DEV_ENVIRONMENT_COMMAND } info`,
+		description: 'Return information about a local dev environment',
+	},
+];
+
+command()
+	.option( 'slug', `Custom name of the dev environment (default: "${ defaults.environmentSlug }")` )
+	.examples( examples )
+	.argv( process.argv, async ( arg, opt ) => {
+		const slug = opt.slug || defaults.environmentSlug;
+
+		debug( 'Args: ', arg, 'Options: ', opt );
+
+		try {
+			await printEnvironmentInfo( slug );
+		} catch ( e ) {
+			let messageToShow = chalk.red( 'Error:' );
```

Comment:
This seems to be repeating for every command now, maybe it would make sense to move this error handling part to a common function?

Reply:
Yeah that's a good point but then I check and the error message I return on stop fail is different by design (no prompt to create environment). So I figured a little bit of copy and paste never heard nobody:) (I was not able to figure out a way to do it in a nice way)